gradle/verification-keyring.keys

```diff
@@ -9519,6 +9458,39 @@ uAngfd6DOtCKYtNKP7xqDu/2bMU23cxGaRj2ToH4RfCClg1B
 =rv9Q
 -----END PGP PUBLIC KEY BLOCK-----
 
+pub    689CBE64F4BC997F
+sub    C0058C509A81C102
```

Comment:
💅 does this key contain a uid?

Reply:
I found one here https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:gradle/verification-keyring.keys;l=9513?authuser=8&hl=zh-cn

I'll add it

Reply:
But it get's removed again automatically =(

Reply:
That's odd, the key itself contains UID packet